conans/client/migrations.py

```diff
@@ -50,3 +52,24 @@ def _apply_migrations(self, old_version):
         # Update profile plugin
         from conans.client.profile_loader import migrate_profile_plugin
         migrate_profile_plugin(cache)
+
+        if old_version and old_version < "2.0.8":
+            _migrate_pkg_db_lru(cache)
+
+
+def _migrate_pkg_db_lru(cache):
```

Comment:
Maybe we should consider the case of one user that install conan 2.0.8, the db is migrated but for some reason that user has to downgrade and then the Conan cache becomes unusable?

Reply:
Yes, I agree. Maybe it's not worth the effort, what worries me about this one is that it may make the whole cache unusable because of a feature you don't event use. Maybe we should think about this, if we want to make the databases retro compatible at least. In this case it would be that if we add a column to the database, previous versions can work with the database that has that extra column but wont use the feature because they don't know about that. Maybe that's something to consider in case we add another feature that modifies the db.

Reply:
I have finally added the copy of the DB to a ``cache.sqlite3.backup`` file, with some visible warnings telling about this file and instructing users to restore it if downgrading to < 2.0.10. 

- The automatic down-migrations is a bit too much at this moment. Seems relatively simple on the surface, but not really
- The python script to down-migrate is possible, but not that straightforward, it is not one sqlite command, but a bit more complicated. I thought that a file copy could be a good compromise.